src/config.ts

```diff
@@ -32,23 +28,16 @@ const schema = {
   },
   userId: {
     type: 'string',
-    default: 'ad6d42d1-1239-526b-a997-6b96888f7719'
-  },
-  'mod-camunda': {
-    type: 'string',
-    default: 'http://localhost:9000'
-  },
-  'mod-camunda-internal': {
-    type: 'string',
-    default: 'http://localhost:9000'
+    default: 'e4252f55-cc7e-5e1e-a17e-7bfc16706a77'
```

Comment:
```suggestion
    default: ''
```
We shouldn't have a default `userId` in here.